src/clipperFunctionalities.cpp

```diff
@@ -0,0 +1,32 @@
+#include <opencv2/core.hpp>
+#include <opencv2/imgproc.hpp>
+#include <opencv2/highgui.hpp>
+
+#include "clipper.hpp"
+
+using namespace ClipperLib;
+
+Paths enlarge(IntPoint* points, int offset){
+    Path subj;
+    Paths solution;
+    for(int i = 0; i < sizeof(points); i++){
+        subj->push_back(points[i]);
+    }
+    ClipperOffset co;
+    co.AddPath(subj, jtRound, etClosedPolygon);
+    co.Execute(solution, offset)
+    return solution;
+}
+
+void printSolution(Paths solution){
+    printf("solution size = %d\n", (int)solutions.size());
+    for(unsigned i = 0; i < solution.size(); i++){
+        Path path = solution.at(i);
+        for(unsigned j = 1; j < path.size(); j++){
+            cv::line(plot, cv::Point2f(path.at(j-1).X, path.at(j-1).Y), cv::Points2f(path.at(j).X, path.at(j).Y), cv::Scalar(255,255,0), 2);
+        }
+    }
+    cv::flip(plot, plot, 0);
```

Comment:
Does it work on the VM?

Reply:
Should work properly, as shown by the professor.

Reply:
It doesn't even compile actually... It's better if we test everything we write. Otherwise we will end up at the end with thousands of bugs to solve